### PR TITLE
setup cppunit and add simple test for modulePackage

### PR DESCRIPTION
**Pull request by mhatina**
To create new tests, you need to create class like:

header file
```
#include <cppunit/TestCase.h>
#include <cppunit/extensions/HelperMacros.h>

class SimpleTest : public CppUnit::TestCase
{

   CPPUNIT_TEST_SUITE(SimpleTest);
        CPPUNIT_TEST(test);
   CPPUNIT_TEST_SUITE_END();

public:
    void setUp() override;
    void tearDown() override;

    void test();
};
```

source file
```
#include "SimpleTest.hpp"

CPPUNIT_TEST_SUITE_REGISTRATION(SimpleTest);

void ModulePackageTest::setUp() {}
void ModulePackageTest::tearDown() {}
void ModulePackageTest::test()
{
    CPPUNIT_ASSERT*();
}
```

and add it into `tests` directory.

### CODE REVIEW

File: libdnf/package/modulepackage.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#include "modulepackage.hpp"
```
Comment (dmach) [inline libdnf/package/modulepackage.cpp:1]: Can you split this into another commit? It has nothing to do with adding tests (AFAIK).

Comment (mhatina) [reply libdnf/package/modulepackage.cpp:1]: Those are methods that are used in tests.
